packages/quantic/force-app/examples/main/lwc/exampleInsightPanel/resultTemplates/parentResultTemplate.html

```diff
@@ -3,15 +3,15 @@
     <c-quantic-result-template
       is-any-preview-open={isAnyPreviewOpen}
       result-preview-should-not-be-accessible={resultPreviewShouldNotBeAccessible}
+      hide-label
     >
-      <c-quantic-result-label
-        slot="label"
-        result={result}
-      ></c-quantic-result-label>
       <div
         slot="badges"
         class="slds-grid slds-grid_vertical-align-center slds-m-vertical_xx-small slds-m-right_small"
       >
+        <c-quantic-colored-result-badge
+          label={result.raw.objecttype}
+        ></c-quantic-colored-result-badge>
```

Comment:
Here you are including the component in the slot `badges`. that's why the it's not positioned as expected.
That's the documentation explaining how the different slots of the Quantic Result Template are positioned: https://docs.coveo.com/en/quantic/latest/usage/result-template-usage/